From 2ee7ce98315864993edaefe49d0e8ba6f9cbf629 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
 <frederic.pierret@qubes-os.org>
Date: Sat, 7 Nov 2020 18:21:07 +0100
Subject: [PATCH] docs/xen-headers: use alphabetical sorting for @incontents

It makes the build reproducible with fileordering flags



-----------
diff --git a/docs/xen-headers b/docs/xen-headers
index 8c434d7..98ffe81 100755
--- a/docs/xen-headers
+++ b/docs/xen-headers
@@ -331,7 +331,7 @@ sub output_index () {
 <h2>Starting points</h2>
 <ul>
 END
-    foreach my $ic (sort { $a->{Seq} <=> $b->{Seq} or $a->{Title} cmp $b->{Title} } @incontents) {
+    foreach my $ic (sort { $a->{Href} cmp $b->{Href} } @incontents) {
         $o .= "<li><a href=\"$ic->{Href}\">$ic->{Title}</a></li>\n";
     }
     $o .= "</ul>\n";
-----------------
